### ISSUE TITLE
Criteria for "certification" of TDWG compliant services

### ISSUE THREAD

**Issue by nickynicolson** [tdwg/apis#62]

**Comment by ben-norton** [comment]
I think the optimal way to establish a TDWG certification is through a set of required best practices. That includes items such as the following:
1. OpenAPI 3.x Compliant API Documentation
2. API is versioned
3. Response Keys in camel-cased format.
4. Appropriate usage of status codes, including a minimum set that must be included  (200, 500, 403, 404, 422)
5. Correct usage of JSON Data Types (e.g., integers are not wrapper in quotation marks)
6. Consistent resource names (e.g., plural/singular)

Here's a good starting place for best practices: https://github.com/opengeospatial/OGC-Web-API-Guidelines

**Comment by ianengelbrecht** [comment]
Agreed with Ben. I don’t think it’s TDWG’s role to be certifying other peoples systems. A guideline is what is needed. The primary reason for developing an API is to make your data or service “to people you’ve never met so they can solve problems you’ve never thought of” — [Mike Ammundsen](https://www.oreilly.com/library/view/restful-web-api/9781098106737/). Presumably if an API uses standards, it will be more user friendly.
